### DISCUSSION TITLE
Quick start with Zigbee on XIAO ESP32C6 by Arduino | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1638] [Q&A]
# Quick start with Zigbee on XIAO ESP32C6 by Arduino | Seeed Studio Wiki

In this tutorial, we will embark on a journey to explore Zigbee application development using the XIAO ESP32C6 development board. The XIAO ESP32C6 is a compact yet powerful board that features the ESP32-C6 chip, which offers integrated Wi-Fi and Bluetooth Low Energy (BLE) connectivity. By leveraging the ESP Zigbee SDK, we can harness the full potential of the XIAO ESP32C6 and extend its capabilities to include Zigbee functionality.

https://wiki.seeedstudio.com/xiao_esp32c6_zigbee_arduino/

**Comment by ms83001** [comment]
Hello, It is a couple of days I 'm trying to make this work on zigbee example. It builds, flash, then resets in loop with nothing happening on concentrator side.
Wifi example works fine.
Building by IDF is also workable.
Is the procedure described on this page confirmed with the latest updates in the Arduino environment ?

Thank you

**Reply by sp-hy** [reply]
Hi I also had this issue and it was driving me mad because using the espressif IDE worked fine.

It was because I had the partition type on the default one, not Zigbee 4mb with spiffs.

**Comment by adisah13** [comment]
Please help me with an example of a sketch in the Arduino IDE 2 of a thermometer running with zigbee protocol using XIAO ESP32C6 and the DS18B20 sensor. I need to read the temperature in the pool. I searched and found just one example of XIAO ESP32C6's chip temperature reading with zigbee protocol
